package.json

```diff
@@ -96,6 +95,7 @@
     "@storybook/preset-typescript": "^3.0.0",
     "@storybook/react": "^5.2.0",
     "@storybook/source-loader": "^5.2.4",
+    "@sumup/collector": "^1.0.0-alpha.1",
```

Comment:
We still want to keep this one as the alpha version?

Reply:
We haven't released a stable version yet 👀